web/docs/Frontend Starters/Next.js/Next.js + Drupal/add-ons.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 'next-drupal-add-ons'
+title: 'Add-ons'
+slug: '/frontend-starters/nextjs/nextjs-drupal/add-ons'
+sidebar_position: 8
+---
+
+## Drupal Umami Demo Data Add-on
+
+This add-on adds necessary components and routes to your Pantheon Decoupled Kit
+Next.js Starter Kit to display data from Drupal's Umami demo site. This includes
+a route for `recipes`, as well as a link in the footer menu to the `recipes`
```

Comment:
@CobyPear also noticed this after the fact. Double check me on this, but I don't think this is completely accurate. The add-on technically doesn't add a footer menu link - that just displays based on the data from Drupal. You could instead say something like 'This includes routes for all individual recipes, along with a index page that displays a grid of all recipes'

Reply:
Yes that sounds more accurate, good catch.